clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3468,33 +3467,31 @@ private boolean dependsOnSpecificNode(ConfigResource resource) {
     }
 
     @Override
-    public MembershipChangeResult removeMemberFromConsumerGroup(String groupId,
-                                                                RemoveMemberFromConsumerGroupOptions options) {
+    public RemoveMembersFromConsumerGroupResult removeMembersFromConsumerGroup(String groupId,
```

Comment:
Tried offline and discovered that the unsupported version exception crashes the network thread instead of populating to the end user. The current conclusion is that we will plan to address this in a separate PR where we also want to add functionality to admin client to surface the exception